internal/pkg/ecs/run_task_request.go

```diff
@@ -271,7 +274,11 @@ func fmtStringMapToString(m map[string]string) string {
 	sort.Strings(keys)
 
 	for _, k := range keys {
-		output = append(output, fmt.Sprintf(`%s="%v"`, k, m[k]))
+		// params are parsed as csv, so we escape double quotes by doubling them up
+		v := strings.ReplaceAll(m[k], `"`, `""`)
+		// need to allow single quotes through the shell escaping
+		v = strings.ReplaceAll(v, `'`, `'\''`)
+		output = append(output, fmt.Sprintf(`"%s=%s"`, k, v))
 	}
-	return strings.Join(output, ",")
+	return `'` + strings.Join(output, ",") + `'`
```

Comment:
I wonder if we can leverage `csv.NewWriter` instead? 
Here is a sample snippet: https://go.dev/play/p/aH4TUM7o4jo

I don't know if this causes any issues with the shell interpreting quotes.

Reply:
ok I've pushed up that change @efekarakus

it's nice that the csv lib only wraps in double quotes when necessary, it cleans up the --generate-cmd output a bit